main.go

```diff
@@ -0,0 +1,342 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	calicocache "github.com/girishshilamkar/k8s-policy-go/pkg/cache"
+	//"github.com/projectcalico/node-controller/pkg/config"
+	"github.com/projectcalico/libcalico-go/lib/api"
+	"github.com/projectcalico/libcalico-go/lib/client"
+	glog "github.com/Sirupsen/logrus"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"k8s.io/apimachinery/pkg/fields"
+	uruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/workqueue"
+	"github.com/kylelemons/godebug/pretty"
+
+)
+
+func main() {
+
+	var kubeconfig string
+	var master string
+
+	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
+	fmt.Println("kubeconfig:",kubeconfig)
+	flag.StringVar(&master, "master", "", "master url")
+	flag.Parse()
+
+	// creates the connection
+	k8sconfig, err := clientcmd.BuildConfigFromFlags(master, kubeconfig)
+	if err != nil {
+		glog.Fatal(err)
+	}
+
+	clientset, err := kubernetes.NewForConfig(k8sconfig)
+	if err != nil {
+		panic(err.Error())
+	}
+	
+	// create the watcher
+	nodeListWatcher := cache.NewListWatchFromClient(clientset.Core().RESTClient(), "namespaces", "", fields.Everything())
+
+	// create the workqueue
+	queue := workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter())
```

Comment:
Do you mean 
**recourse cache** can then expose an interface which can be called to get the output queue. ?